fortnite_api/stats.py

```diff
@@ -1,65 +1,261 @@
+        The player's stats for gamepad input. This is ``None`` if the player has no stats.
+    touch: Optional[:class:`BrInputStats`]
+        The player's stats for touch input. This is ``None`` if the player has no stats.
+    """
+
+    __slots__: Tuple[str, ...] = ('all', 'keyboard_mouse', 'gamepad', 'touch')
+
+    def __init__(self, *, data: Dict[str, Any]):
+
+        _all = data.get('all')
+        self.all: Optional[BrInputStats] = _all and BrInputStats(data=_all)
+
+        _keyboard_mouse = data.get('keyboardMouse')
+        self.keyboard_mouse: Optional[BrInputStats] = _keyboard_mouse and BrInputStats(data=_keyboard_mouse)
+
+        _gamepad = data.get('gamepad')
+        self.gamepad: Optional[BrInputStats] = _gamepad and BrInputStats(data=_gamepad)
+
+        _touch = data.get('touch')
+        self.touch: Optional[BrInputStats] = _touch and BrInputStats(data=_touch)
 
 
 class BrInputStats:
+    """Represents a specific Fortnite player's stats for a specific input type.
+
+    Attributes
+    ----------
+    overall: Optional[:class:`BrGameModeStats`]
+        The overall stats for the player. This is ``None`` if the overall stats are not available.
+    solo: Optional[:class:`BrGameModeStats`]
+        The player's stats for solo game modes. This is ``None`` if the player has no stats for solo game modes.
+    duo: Optional[:class:`BrGameModeStats`]
+        The player's stats for duo game modes. This is ``None`` if the player has no stats for duo game modes.
+    trio: Optional[:class:`BrGameModeStats`]
+        The player's stats for trio game modes. This is ``None`` if the player has no stats for trio game modes.
+    squad: Optional[:class:`BrGameMode`]
+        The player's stats for squad game modes. This is ``None`` if the player has no stats for squad game modes.
+    """
+
+    __slots__: Tuple[str, ...] = ('overall', 'solo', 'duo', 'trio', 'squad', 'raw_data')
+
+    def __init__(self, *, data: Dict[str, Any]) -> None:
+        _overall = data.get('overall')
+        self.overall: Optional[BrGameModeStats] = _overall and BrGameModeStats(data=_overall)
 
-    def __init__(self, data):
-        self.overall = BrGameModeStats(data.get('overall')) if data.get('overall') else None
-        self.solo = BrGameModeStats(data.get('solo')) if data.get('solo') else None
-        self.duo = BrGameModeStats(data.get('duo')) if data.get('duo') else None
-        self.trio = BrGameModeStats(data.get('trio')) if data.get('trio') else None
-        self.squad = BrGameModeStats(data.get('squad')) if data.get('squad') else None
-        self.raw_data = data
+        _solo = data.get('solo')
+        self.solo: Optional[BrGameModeStats] = _solo and BrGameModeStats(data=_solo)
+
+        _duo = data.get('duo')
+        self.duo: Optional[BrGameModeStats] = _duo and BrGameModeStats(data=_duo)
+
+        _trio = data.get('trio')
+        self.trio: Optional[BrGameModeStats] = _trio and BrGameModeStats(data=_trio)
```

Comment:
We probably can remove this, since trio news have been removed in favour of representing stats as it is in-game. This field is always empty and just exists as a legacy.